collect_app/src/main/java/org/odk/collect/android/widgets/video/ExVideoWidgetContent.kt

```diff
@@ -0,0 +1,62 @@
+package org.odk.collect.android.widgets.video
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.automirrored.filled.OpenInNew
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import org.odk.collect.android.widgets.WidgetIconButton
+import org.odk.collect.strings.R.string
+
+@Composable
+fun ExVideoWidgetContent(
+    readOnly: Boolean,
+    fontSize: Int,
+    onLaunchClick: () -> Unit,
+    onLongClick: () -> Unit,
+    widgetAnswer: @Composable () -> Unit
+) {
+    Column {
+        if (!readOnly) {
+            WidgetIconButton(
+                Icons.AutoMirrored.Filled.OpenInNew,
+                stringResource(string.launch_app),
+                fontSize,
+                onLaunchClick,
+                onLongClick
+            )
+        }
+
+        widgetAnswer()
```

Comment:
One quick tweak that we could make here and in `VideoWidgetContent`: you could place this call in a `Box` (or some other simple container) that adds the margin so that can be a concern of the layout code here rather than in the passed Composable declaration.